Getting_and_Cleaning_Data/Tidying_Data_with_tidyr/students3.csv

```diff
@@ -2,7 +2,7 @@ name,test,class1,class2,class3,class4,class5
 Sally,midterm,A,,B,,
 Sally,final,C,,C,,
 Jeff,midterm,,D,,A,
-Jeff,final,,E,,C,
```

Comment:
I don't think that changing the grades from E to F is really necessary since swirl's audience is international, and E is a letter grade commonly used in India, Pakistan, South Korea, and [many other countries](http://en.wikipedia.org/wiki/Grading_systems_by_country) where there are a significant number of swirl users.